feedhenry/fh-js-sdk
# fix(SDK broken with client forms app): Downgrade dep libs to be the same of the 3.0.2 version released

# Jira link(s)
https://issues.jboss.org/browse/RHMAP-21479

# What
Downgrade dep libs to be the same of the 3.0.2 version released

# Why
The upper versions of browserfly is broken the client forms app

# How
Use the same dev and browserfly libs used in 3.0.2

# Verification Steps
- Clone the app https://github.com/feedhenry/Appforms-Template-v3
- Save the 3.0.9 tag development of this project
- Run npm install
- Run grunt
- Go to RHMAP and create a new form project
- Link this form project to a form
- Deploy and start the cloud app
- See in the preview the client app working
- Run grunt serve:local -url {clouURL}

## Checklist:
